bridge/elrond/client.go

```diff
@@ -189,13 +190,21 @@ func parseIntFromByteSlice(buff []byte) (int64, error) {
 }
 
 // ProposeSetStatus will trigger the proposal of the ESDT safe set current transaction batch status operation
-func (c *client) ProposeSetStatus(_ context.Context, batch *bridge.Batch) {
+func (c *client) ProposeSetStatus(ctx context.Context, batch *bridge.Batch) {
 	builder := newBuilder(c.log).
 		Func("proposeEsdtSafeSetCurrentTransactionBatchStatus").
 		BatchId(batch.Id)
 
-	for _, tx := range batch.Transactions {
-		builder = builder.Int(big.NewInt(int64(tx.Status)))
+	newBatch, err := c.GetPending(ctx)
+	if err != nil {
+		c.log.Error("Elrond: get pending batch failed in ProposeSetStatus", "error", err.Error())
```

Comment:
optional: could use `err` instead of `err.Error()`

Reply:
done